README.md

```diff
@@ -173,6 +173,16 @@ const hat = await MakeHat({ inches: 12 }, { baseURL: "http://localhost:8080" });
 console.log(hat);
 ```
 
+You might want to configure baseURL globally for all API calls:
+```ts
+import { client } from "twirpscript";
+client.baseURL = "http://localhost:8080";
+
+const = await MakeHat({ inches: 12 }); // We can omit the baseURL config since it's already set
+
+```
```

Comment:
```suggestion
`baseURL` can be configured globally, instead of providing it for every RPC call site:

```ts
import { client } from "twirpscript";
client.baseURL = "http://localhost:8080";

const hat = await MakeHat({ inches: 12 }); // We can omit `baseURL` because it has already been set
console.log(hat);

const hat = await MakeHat({ inches: 12 }, { baseURL: "https://api.example.com"); // We can also override the globally configured `baseURL` with a different value
console.log(hat);
```

In addition to `baseURL`, `headers` may also be configured globally, and/or optionally override at the RPC call site. The key value pairs in `headers` will be supplied as HTTP headers on every API request.

```ts
import { client } from "twirpscript";
client.baseURL = "http://localhost:8080";
client.headers = { "device-id": "some-device-id" }

const hat = await MakeHat({ inches: 12 });
console.log(hat);
```

```